### PR TITLE
Add Req 10 test

### PR DESCRIPTION
**Pull request by JasminMus**
**Pull Request Requirements:**

- Provide a brief description of the changeset.
- Include a screenshots if applicable
- Ensure that the changeset adheres to the [DCO guidelines](https://github.com/apps/dco).

### CODE REVIEW

File: qms-tests/sanity/tests/documents/REQ-10.spec.ts [hunk 1]
```diff
@@ -75,4 +75,53 @@ test.describe('ISO 13485, 4.2.4 Control of documents ensure that documents of ex
...
+
+  test('TESTS-404. As a space member only, I cannot create any doc from that space', async ({ page }) => {
+    await allure.description('Requirement\nUser is not able to create any document from that space')
+    await allure.tms('TESTS-404', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-404')
+    await test.step('2. cCheck if user can not create documents as a space member', async () => {
+      const folderName = faker.word.words(1)
+      const documentContentPage = new DocumentContentPage(page)
+      await documentContentPage.clickAddFolderButton()
+      await documentContentPage.createDocumentSpaceMembersToJustMember(folderName)
+      await documentContentPage.checkIfUserCanSelectSpace(folderName, false)
```
Comment (lexiv0re) [inline qms-tests/sanity/tests/documents/REQ-10.spec.ts:126]: There are multiple ways to add new documents to a space. New document button (wizard) is only one of them. Other ways are by adding a new document of the space or child document of a document in the space from the context menu of the space/document respectively. To check the requirement we should check that all of these options are not available.

File: qms-tests/sanity/tests/model/documents/document-content-page.ts [hunk 2]
```diff
@@ -404,6 +453,19 @@ export class DocumentContentPage extends DocumentCommonPage {
     await this.checkDocumentStatus(DocumentStatus.DRAFT)
   }
 
+  async checkTeamMembersReviewNotExists (): Promise<void> {
```
Comment (lexiv0re) [inline qms-tests/sanity/tests/model/documents/document-content-page.ts]: Minor: the naming is confusing as it checks not only review but approver/co-author assignment as well.

Comment (JasminMus) [reply qms-tests/sanity/tests/model/documents/document-content-page.ts]: hmmm true, I will rename it

File: qms-tests/sanity/tests/documents/REQ-10.spec.ts (hunk 1 above)
Comment (JasminMus) [reply qms-tests/sanity/tests/documents/REQ-10.spec.ts:126]: I followed the steps from the TC, but I can update it so we can see that user is not able to add document through other methods as well.

**Review by lexiv0re — APPROVED** [review]

**Pull request merged**